assignments/function-conversion.js

```diff
@@ -5,21 +5,35 @@
 // };
 // myFunction();
 
+let myfunction = () => {};
```

Comment:
The original function did a console log. Try having your arrow function version do that as well.